test/tup.sh

```diff
@@ -247,10 +247,10 @@ __update()
 	fi
 
 	set +e
-	if [ -z $@ ]; then
-		$cmd "$@"
+	if [ $# -le 0 ]; then
```

Comment:
What are the changes to this file supposed to fix? All tests passed without them on my machines. Is this a separate issue that we need to address for Gentoo?